eng/pipelines/common/templates/run-test-job.yml

```diff
@@ -20,14 +20,15 @@ parameters:
   runInUnloadableContext: false
```

Comment:
Could we put these files under `/eng/pipelines/common/templates/runtimes/*` or something like that so that the path is explicit that these templates are shared in between runtimes?